From 6adae690dd708d94d9d3028d086c257903b58d11 Mon Sep 17 00:00:00 2001
From: Shwetha B <shwetha.b@intel.com>
Date: Tue, 9 Feb 2021 10:26:40 +0530
Subject: [PATCH] Selinux for power related system property

Assign exported_default_prop for the system property
ro.power.fixed_performance_scale_factor

Tracked-On:
Signed-off-by: Shwetha B <shwetha.b@intel.com>
---
 prebuilts/api/30.0/public/property_contexts | 3 +++
 public/property_contexts                    | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/prebuilts/api/30.0/public/property_contexts b/prebuilts/api/30.0/public/property_contexts
index 4607ef325..aa24f2f18 100644
--- a/prebuilts/api/30.0/public/property_contexts
+++ b/prebuilts/api/30.0/public/property_contexts
@@ -477,3 +477,6 @@ cache_key.telephony.                     u:object_r:binder_cache_telephony_serve
 # Graphics related properties
 graphics.gpu.profiler.support          u:object_r:graphics_config_prop:s0 exact bool
 graphics.gpu.profiler.vulkan_layer_apk u:object_r:graphics_config_prop:s0 exact string
+
+#Power related properties
+ro.power.fixed_performance_scale_factor u:object_r:exported_default_prop:s0 exact int
diff --git a/public/property_contexts b/public/property_contexts
index 4607ef325..aa24f2f18 100644
--- a/public/property_contexts
+++ b/public/property_contexts
@@ -477,3 +477,6 @@ cache_key.telephony.                     u:object_r:binder_cache_telephony_serve
 # Graphics related properties
 graphics.gpu.profiler.support          u:object_r:graphics_config_prop:s0 exact bool
 graphics.gpu.profiler.vulkan_layer_apk u:object_r:graphics_config_prop:s0 exact string
+
+#Power related properties
+ro.power.fixed_performance_scale_factor u:object_r:exported_default_prop:s0 exact int
-- 
2.29.2

